src/utils/payments/pawapay/pawapay.ts

```
@@ -0,0 +1,473 @@
+      }
+    } catch (error) {
+      console.error('Error resending payout callback:', error);
+      throw error;
+    }
+  }
+
+  // Validate a callback signature
+  validateCallbackSignature(
+    headers: Record<string, string>,
+    body: any,
+    publicKey: string
```

ESLint

[error] 'publicKey' is defined but never used. Allowed unused args must match /^_/u.

(unused-imports/no-unused-vars)